tachyon/zk/plonk/keys/assembly.h

```diff
@@ -38,6 +38,7 @@ class Assembly : public Assignment<typename PCS::Field> {
   const std::vector<RationalEvals>& fixed_columns() const {
     return fixed_columns_;
   }
+  PermutationAssembly<PCS>& permutation() { return permutation_; }
```

Comment:
Why did you add this?

Reply:
Since `CommitLagrange` in univariate_polynomial_commitment_scheme.h is not marked as `const`, `BuildVerifyingKey` in permutation_assembly.h is not marked as `const`. 
So, when doing 
```c++
permutation_verifying_Key_ =
        pre_load_result.assembly.permutation().BuildVerifyingKey(entity,
                                                                 permutations);
```
in verifying_key.h, we need ref getter, so I added it.

Reply:
I still don't get it. Could you tell me at office?

Reply:
`BuildVerifyingKey()` doesn't have to be mutable right?